src/app/models/trip.js

```diff
@@ -0,0 +1,33 @@
+import Backbone from 'backbone';
+
+const Trip = Backbone.Model.extend({
+  // These are properties!
+  urlRoot: 'https://ada-backtrek-api.herokuapp.com/trips',
+  validate: function(attributes) {
+    const errors = {};
+
+    if (!attributes.name) {
+      errors['name'] = ['Name cannot be blank!'];
+    }
+    if (!attributes.continent) {
+      errors['continent'] = ['Continent cannot be blank!'];
+    }
+    if (!attributes.category) {
+      errors['category'] = ['Category cannot be blank!'];
+    }
+    if (!attributes.weeks) {
```

Comment:
I would also check for a numeric value of weeks and cost and make sure they are non-negative.